### PR TITLE
Stream-It Movie Database Project - Version One

### PR DESCRIPTION
**Pull request by Benmuiruri**
### Javascript Capstone Project  

**Devs - @Benmuiruri  and [Ola](https://github.com/olawalecoder)**

**Tech/Techniques used**

- ES6 syntax.
- ES6 modules.
- callbacks and promises.
- Webpack.
- AAA pattern for unit tests.
- Write units tests for a JavaScript app.
- Follow Gitflow.
- Solve simple git conflicts.
- Send and receive data from an API.
- Use API documentation.
- Here's a round-up of the features that were implemented in this project

**API Used**

- Movie API provided by [TV Maze](https://www.tvmaze.com/api)
- Involvement API provided by **Microverse** 

**Website Features**

- A home page showing a list of Movies that you can "like."
- A popup window with more data about a movie
- Comment feature inside the modal to add new comments and view comments for a particular movie
- Counter showing the number of movies displayed on the home page, the number of likes under the movie title, and the number of comments in the popup window.

**Link to video presentation**
- https://drive.google.com/file/d/1YSgEySc4y8TBjSsUmKJma8wksQKKW5Ii/view?usp=sharing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,56 @@
...
+
+<p align="center">Desktop view</p>
+
+## Preview
+![Preview](./src/assets/streamdsktphmpage.png)
+![Preview](./src/assets/st-desk-1.png)
+![Preview](./src/assets/st-desk-2.png)
+
+<h1 align="center"> Stream-It </h1>
+
```
Comment (Rustamxon7) [inline README.md:11]: **You made a great effort on this project.** 
_However, some changes need to be completed to make it approvable!_

- [ ] I cannot see your presentation video about this project. Please add a link to the readme file about your presentation.

> HINTS: 
The student recorded a video presenting the features of the project.
The maximum length of the video is 5 minutes.
The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
 All the members in the video should appear in the video and talk about the project.
The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

_I would suggest using Zoom to record the video!_

**Review by Rustamxon7 — CHANGES_REQUESTED** [review]
Hello TEAM, 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Benmuiruri) [reply README.md:11]: Hi @Rustamxon7  thanks for the review, I have added the video link to the README.

**Review by MrBrN197 — COMMENTED** [review]
## Hi, @Benmuiruri   👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Benmuiruri** [comment]
Hi @MrBrN197 thanks for the review and the suggested changes as soon as possible.

**Comment by Benmuiruri** [comment]
I addressed the issues in this PR #38

**Pull request merged**